app/models/exchange.rb

```diff
@@ -99,21 +99,6 @@ def participant
     incoming? ? sender : receiver
   end
 
-  def to_h(core_only = false)
-    h = attributes.merge('variant_ids' => variant_ids.sort,
-                         'enterprise_fee_ids' => enterprise_fee_ids.sort)
-    h.reject! { |k| %w(id order_cycle_id created_at updated_at).include? k } if core_only
-    h
-  end
-
-  def eql?(e)
```

Comment:
This is awesome. :fire: :joy: I'm sure this took a lot of courage @mkllnk.

Reply:
Yes, a little bit. How do you find out what code is affected by this change?

- There was no usage of `eql?`.
- There was no usage of `uniq` on anything that looked like exchanges.

`uniq` is mentioned in the Ruby documentation of `eql?`. So I was hoping that any other core method using `eql?` would be mentioned there as well. Hence, there should be no other code using it. :crossed_fingers: